### PR TITLE
fixed format content in beehive/pkg/core/context/context_channel.go

### PR DESCRIPTION
**Pull request by orangegzx**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
>The initial letter of the content of the formatted string is not capitalized uniformly

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by gy95** [comment]
the error fmt commonly starts with a lowercase letter, more readable when print errors to logs. 😄 🥲

File: staging/src/github.com/kubeedge/beehive/pkg/core/context/context_channel.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func (ctx *ChannelContext) Receive(module string) (model.Message, error) {
 	}
 
 	klog.Warningf("Failed to get channel for module:%s when receive message", module)
-	return model.Message{}, fmt.Errorf("failed to get channel for module(%s)", module)
+	return model.Message{}, fmt.Errorf("Failed to get channel for module(%s)", module)
```
Comment (zc2638) [inline staging/src/github.com/kubeedge/beehive/pkg/core/context/context_channel.go:83]: The first letter of the `log` output needs to be capitalized, but the `error` not recommended

Comment (fisherxu) [reply staging/src/github.com/kubeedge/beehive/pkg/core/context/context_channel.go:83]: +1 :)

**Comment by gy95** [comment]
when use `klog.Errorf` or `klog.Infof` etc. to print logs, we can use capitalized string.
But when we use `fmt.Errorf` or error.New to format error info, msg should start with lowercase

File: staging/src/github.com/kubeedge/beehive/pkg/core/context/context_channel.go [hunk 2]
```diff
@@ -262,11 +262,11 @@ func (ctx *ChannelContext) SendToGroupSync(moduleType string, message model.Mess
 		case <-sendTimer.C:
 			cleanup()
 			if timeoutCounter != 0 {
-				errInfo := fmt.Sprintf("timeout to send message, several %d timeout when send", timeoutCounter)
+				errInfo := fmt.Sprintf("Timeout to send message, several %d timeout when send", timeoutCounter)
 				return fmt.Errorf(errInfo)
```
Comment (zc2638) [inline staging/src/github.com/kubeedge/beehive/pkg/core/context/context_channel.go:266]:
```suggestion
				return fmt.Errorf(Timeout to send message, several %d timeout when send", timeoutCounter)
```
It is recommended to combine the two methods

File: staging/src/github.com/kubeedge/beehive/pkg/core/context/context_channel.go [hunk 3]
```diff
@@ -262,11 +262,10 @@ func (ctx *ChannelContext) SendToGroupSync(moduleType string, message model.Mess
 		case <-sendTimer.C:
 			cleanup()
 			if timeoutCounter != 0 {
-				errInfo := fmt.Sprintf("timeout to send message, several %d timeout when send", timeoutCounter)
-				return fmt.Errorf(errInfo)
+				return  fmt.Errorf("Timeout to send message, several %d timeout when send", timeoutCounter)
```
Comment (gy95) [inline staging/src/github.com/kubeedge/beehive/pkg/core/context/context_channel.go:265]: need to fixed from `Timeout` to `timeout`

**Review by fisherxu — COMMENTED** [review]
/approve
Seems that there's many redundant logs, as first klog print the error, then also return the error, then print it  again.

**Pull request merged**